frescobaldi_app/job/lilypond.py

```diff
@@ -173,8 +173,18 @@ def configure_command(self):
         # just before starting the process.
         if sys.platform.startswith('darwin'):
             import macosx
-            if macosx.inside_app_bundle() and self.lilypond_info.frommacports():
-                self.environment['PATH'] = self.lilypond_info.bindir()
+            if macosx.inside_app_bundle():
+                bindir = self.lilypond_info.bindir()
+                if self.lilypond_info.frommacports():
+                    self.environment['PATH'] = bindir
+                else:
+                    # Add PATH for Homebrew.
+                    import re
+                    regexp = re.compile('(.*)/Cellar/.*')
+                    prefix = regexp.match(bindir)
+                    if prefix:
+                        bindir = prefix[1] + '/bin'
```

Comment:
Even better, can you directly fix `lilypondinfo.bindir` (using `fromhomebrew` and these lines therein) and here simply do
```
if macosx.inside_app_bundle() and (self.lilypond_info.frommacports() or self.lilypond_info.fromhomebrew()):
                self.environment['PATH'] = self.lilypond_info.bindir()
```
?

Reply:
I had initially tried to get Frescobaldi working by adding `/usr/local/bin/lilypond` in the settings. That's a symbolic link to `../Cellar/lilypond/2.22.1/bin/lilypond`, and Frescobaldi used `/usr/local/Cellar/lilypond/2.22.1/bin` for `self.lilypond_info.bindir()` which is not the right path to find `gs`. So that case must be handled correctly as well.

My code should work fine with the app bundle for lilypond, too, and fix also manual installations which often install `lilypond` and `gs` in `/usr/local/bin`, so it is not restricted to Homebrew.

Reply:
That's a good idea and might work, but then we have to make sure that symbolic links in the `lilypond` path are not resolved, because the resolved path is no longer valid for finding `gs`.